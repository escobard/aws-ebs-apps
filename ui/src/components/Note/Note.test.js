import React from "react";
import { shallow } from "enzyme";
import Note from "./index";

describe("Form snapshot renders", () => {
  it("should render note component", () => {
    const note = {
      title: "Test title",
      note:
          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vel nulla sit amet nibh sagittis eleifend. Cras a lacus rutrum ipsum pretium scelerisque sed eu turpis. ",
      date: "9 am"
    };
    const component = shallow(<Note data={note} />);
    
    expect(component).toMatchSnapshot();
  });
});
