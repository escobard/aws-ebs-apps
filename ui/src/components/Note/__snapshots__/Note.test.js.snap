// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form snapshot renders should render makeDonation form and fields correctly 1`] = `
<Grid
  columns="equal"
  id="undefined-note"
>
  <GridColumn>
    <i
      aria-hidden="true"
      className="sticky note big circular icon"
    />
  </GridColumn>
  <GridColumn
    className="content"
    computer={14}
    mobile={9}
    tablet={12}
  >
    <h3 />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vel nulla sit amet nibh sagittis eleifend. Cras a lacus rutrum ipsum pretium scelerisque sed eu turpis. 
    </p>
  </GridColumn>
  <GridColumn
    className="date"
  >
    <p>
      9 am
    </p>
  </GridColumn>
</Grid>
`;
